### ISSUE TITLE
Ramdisk not can't load when I try to manually install. 

### ISSUE THREAD

**Issue by hammish55** [Wack0/maciNTosh#29]
When I try to load Ramdisk I get a error message " Failed to load RAMDISK : Invalid  device name. " Any suggestions? Excuse my noob question.

**Comment by Wack0** [comment]
Is the cd drive showing up in the device list?

**Comment by hammish55** [comment]
Not sure how to view the device list.  Please explain.

**Comment by hammish55** [comment]
I see the Hard drive and the CD drive after

**Comment by hammish55** [comment]
Any help to resolve this would me most helpful

**Comment by Wack0** [comment]
Does the same error show up every time you try to load the ramdisk?

What optical drive are you using?

**Comment by hammish55** [comment]
Yes it does. I let NT run then it says can't find machine type. So from the list I go to OTHER and I press the enter button and it asks again. And repeats asking. I am using the OEM Apple cd-rom drive. I was thinking of swapping out to a more modern dvd-rw drive for testing.  Also from the device list I never see the RAMDISK as being loaded.

**Comment by Wack0** [comment]
The invalid device name error would be consistent with the IDE/ATAPI driver failing to open the drive (so test unit ready/start unit/read capacity commands failing)... maybe I should look at open firmware's atapi driver to see what it does...

That said, the only Apple OEM optical drive I'm familiar with is the one that came with my Lombard which I had to replace due to it being unable to read parts of discs... it did work otherwise fine inside the ARC firmware though...

**Comment by Wack0** [comment]
OK, so when working on the uni-north port I found a slight issue with the ide driver, specifically the atapi side, on every access it sends test unit ready + start/stop unit (start) + read capacity, some drives dislike the repeated start/stop unit. The one in my snow will just spin back down when getting sent that, causing all disc reads to be very slow, I wouldn't be surprised if it causes other issues on other drives.

I will port the fix to grackle arcfw when I get time...
